components/Header.js

```diff
@@ -0,0 +1,25 @@
+import { data } from '../data.js';
+
+export function createHeader() {
+  const { header: {headerTitle, teamTitle, teamDescription} } = data; 
+  const headerElement = document.createElement('header');
+  headerElement.className = 'header-container';
+
+  const crewTittle = document.createElement('h1');
+  crewTittle.className = 'header-title';
+  crewTittle.textContent = headerTitle;
```

Comment:
I'm not sure this is working as expected. Is the `textContent` value being displayed? could you share a screenshot here to see how it looks?